src/script/script.h

```diff
@@ -495,65 +497,14 @@ class CScript : public CScriptBase
 
     bool GetOp(const_iterator& pc, opcodetype& opcodeRet, std::vector<unsigned char>& vchRet) const
     {
-        return GetOp2(pc, opcodeRet, &vchRet);
+        return GetScriptOp(pc, end(), opcodeRet, &vchRet);
     }
 
     bool GetOp(const_iterator& pc, opcodetype& opcodeRet) const
     {
-        return GetOp2(pc, opcodeRet, nullptr);
+        return GetScriptOp(pc, end(), opcodeRet, nullptr);
     }
```

Comment:
Could also move the `GetOp` definitions and make `GetScriptOp` static.

Reply:
That defeats the purpose. Longer term is needs to be accessible from the interpreter code.